src/backend/gl/src/device.rs

```diff
@@ -515,7 +514,6 @@ impl d::Device<B> for Device {
 
             Ok(n::Memory {
                 properties: memory::Properties::CPU_VISIBLE
-                    | memory::Properties::COHERENT
```

Comment:
I'm sorry to bring it up, but it looks like we should expose both coherent and non-coherent memory types for CPU visible buffers (where persistent mapping is available), given that GL has the corresponding `MAP_FLUSH_EXPLICIT_BIT` bit that we can control.

It is totally fine to delay for later. My only concern would be that some of gfx-hal users may expect at least one coherent memory type exposed, since this is a guarantee of Vulkan.

Reply:
I can add that as well.

Reply:
done (in personal branch), waiting on glow.